driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -486,6 +487,7 @@ private void maybeInitCluster(){
                 try {
                     //launch ccm cluster
                     ccmBridge = CCMBridge.create("test-class");
+                    ccmBridge.updateConfig("enable_user_defined_functions", "true");
```

Comment:
It looks like that will indeed break on older versions:

    38947  [Exec Stream Pumper] ERROR com.datastax.driver.core.CCMBridge  - ccmerr> Error starting node node1
    38948  [Exec Stream Pumper] ERROR com.datastax.driver.core.CCMBridge  - ccmerr> Standard error output is:
    38948  [Exec Stream Pumper] ERROR com.datastax.driver.core.CCMBridge  - ccmerr> Invalid yaml. Please remove properties [enable_user_defined_functions] from your cassandra.yaml
    38948  [Exec Stream Pumper] ERROR com.datastax.driver.core.CCMBridge  - ccmerr> Fatal configuration error; unable to start. See log for stacktrace.